# Allocation of robots and data they are responsible for getting
# vehicle_idx: [0, 1]
vehicle_idx: [0, 1, 2]
# data_idx: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
data_idx: [0, 1, 2, 3, 4, 5, 6]
# data_idx: [0, 1]


